wp-graphql-smart-cache.php

```diff
@@ -75,7 +75,8 @@ function can_load_plugin() {
 	}
 
 	// Have we met the minimum version requirement?
-	if ( true === version_compare( WPGRAPHQL_VERSION, WPGRAPHQL_SMART_CACHE_WPGRAPHQL_REQUIRED_MIN_VERSION, 'lt' ) ) {
+	// @phpstan-ignore-next-line
```

Comment:
Is this because the constant `WPGRAPHQL_VERSION` doesn't exist?

Reply:
Kinda.  Because it is parsed from PHPDoc.  Actual phpstan message is this

```
If condition is always false.                                       
         💡 Because the type is coming from a PHPDoc, you can turn off this  
            check by setting treatPhpDocTypesAsCertain: false in your        
            phpstan.neon.dist.                                         
```